test/unit/reports/get-events.test.js

```diff
@@ -0,0 +1,76 @@
+const { PAYMENT_EVENT } = require('../../../app/constants/event-types')
+const { stringifyEventData } = require('../../helpers/stringify-event-data')
+
+jest.mock('../../../app/storage')
+const { getClient: mockGetClient, odata: mockOdata } = require('../../../app/storage')
+
+const mockListEntities = jest.fn()
+
+const mockTableClient = {
+  listEntities: mockListEntities
+}
+
+let extractedEvent
+let enrichedEvent
+let events
+
+const { getEvents } = require('../../../app/reports/mi-report/get-events')
+
+describe('get events', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+
+    extractedEvent = JSON.parse(JSON.stringify(require('../../mocks/events/extracted')))
+    enrichedEvent = JSON.parse(JSON.stringify(require('../../mocks/events/enriched')))
+
+    stringifyEventData(extractedEvent)
+    stringifyEventData(enrichedEvent)
+
+    events = [extractedEvent, enrichedEvent]
+
+    mockGetClient.mockReturnValue(mockTableClient)
+    mockListEntities.mockReturnValue(events)
+  })
+
+  test('should get payment client', async () => {
+    await getEvents()
+    expect(mockGetClient).toHaveBeenCalledTimes(1)
+  })
+
+  test('should get payment client once', async () => {
+    await getEvents()
+    expect(mockGetClient).toHaveBeenCalledTimes(1)
+  })
+
+  test('should get payment client with payment event type', async () => {
+    await getEvents()
+    expect(mockGetClient).toHaveBeenCalledWith(PAYMENT_EVENT)
+  })
+
+  test('should get payment events once', async () => {
```

Comment:
Do we need a test for 'should get payment events'?

Reply:
What would it add that this doesn't?